admin/client/src/components/MessageBox/MessageBox.js

```diff
@@ -0,0 +1,105 @@
+import React, { PropTypes } from 'react';
+import SilverStripeComponent from 'lib/SilverStripeComponent';
+import { Alert } from 'react-bootstrap-ss';
+import castStringToElement from 'lib/castStringToElement';
+
+/**
+ * A wrapper for Alert messages in react-bootstrap.
+ * Displays a given message.
+ */
+class MessageBox extends SilverStripeComponent {
```

Comment:
Yeah, I had used `MessageBox` as a placeholder, didn't give a lot of thought about a proper name, `AlertBox` was my alternative.
Anyway, `Alert` can be used instead.